Sources/ApodiniHTTP/InterfaceExporter.swift

```diff
@@ -54,57 +53,71 @@ public final class HTTP: Configuration {
 
 struct Exporter: InterfaceExporter {
     let app: Apodini.Application
-    
     let configuration: HTTP.ExporterConfiguration
-    
     let logger: Logger
     
     init(_ app: Apodini.Application, _ configuration: HTTP.ExporterConfiguration) {
         self.app = app
         self.configuration = configuration
         self.logger = app.logger
-        
-        // Set option to activate case insensitive routing, default is false (so case-sensitive)
-        self.app.vapor.app.routes.caseInsensitive = configuration.caseInsensitiveRouting
+        // Note: a thing to consider here is that we might want to have per-route case sensitivity.
+        // For example, if the REST IE wants case-insensitive routing, but the HTTP IE does not, that'd somehow need to be supported.
```

Comment:
Is this currently supported, not right?

Reply:
In the Vapor-based version? No, not that I'm aware. There's only one Vapor application, and if multiple IEs set a property on that application's http config, the last one wins.